tools/gce_setup/grpc_docker.sh

```diff
@@ -797,7 +797,26 @@ grpc_interop_test() {
   echo "  $ssh_cmd"
   echo "on $host"
   [[ $dry_run == 1 ]] && return 0  # don't run the command on a dry run
-  gcloud compute $project_opt ssh $zone_opt $host --command "$cmd" 
+  gcloud compute $project_opt ssh $zone_opt $host --command "$cmd" &
```

Comment:
A little comment that describes what this is doing would be nice. Also, you have virtually the same logic here twice, so how about providing a function that does the waiting and killing magic and and a docline for it?